src/Sylius/Bundle/CoreBundle/Validator/Constraints/OneEnabledEntityValidator.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
```

Comment:
IMHO I would put it to `Sylius\Bundle\CoreBundle\Validator` since it isn't constraint.